ml-agents/mlagents/trainers/torch/layers.py

```diff
@@ -206,7 +207,16 @@ def forward(
         # We don't use torch.split here since it is not supported by Barracuda
         h0 = memories[:, :, : self.hidden_size].contiguous()
         c0 = memories[:, :, self.hidden_size :].contiguous()
+
+        if exporting_to_onnx.is_exporting():
```

Comment:
Is the comment above about `torch.split` still accurate?

Reply:
Yes, this corresponds to a slice operator, not a split.

Reply:
I actually think `split` was added to recent versions of Barracuda, and we could use it here if we're breaking compat with old versions anyway. https://docs.unity3d.com/Packages/com.unity.barracuda@1.3/manual/SupportedOperators.html#Split